src/PostgREST/RangeQuery.hs

```diff
@@ -38,12 +41,13 @@ rangeParse range = do
       rangeIntersection lower upper
     Nothing -> rangeGeq 0
 
-rangeRequested :: RequestHeaders -> NonnegRange
-rangeRequested = rangeParse . fromMaybe "" . lookup hRange
+rangeRequested :: RequestHeaders -> Maybe NonnegRange
```

Comment:
The Maybe ties into this function https://github.com/begriffs/postgrest/pull/608/files#diff-02f231cf09412413f8f0e9d5e5d7a06eR193
and i use Nothing to "not set" the limit for the top level range if it's not specified, so if there is no value for it, there is actually nothing in the sql, before this, it used to be "Limit all"